lib/browser/_asserters.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+var util = require('util');
+
+var assert = require('assertive');
+var Bluebird = require('bluebird');
+var _ = require('lodash');
+var wd = require('wd');
+
+var matchers = require('./_matchers');
+
+var Asserter = wd.Asserter;
+
+function AssertionError() {
+  Error.call(this);
+  this.message = util.format.apply(util, arguments);
+  this.retriable = true;
+
+  Error.captureStackTrace(this, AssertionError);
+}
+util.inherits(AssertionError, Error);
+
+exports.isDisplayed = function assertIsDisplayed(expected, selector) {
+  return new Asserter(function testTarget(target) {
+    if (!target) {
+      throw new AssertionError('Element not found for selector: ' + selector);
+    }
+
+    return target.isDisplayed().then(function compareTo(actual) {
+      if (actual !== expected) {
+        throw new AssertionError('Element %j should%s be displayed',
+          selector, (expected ? '' : 'n\'t'));
+      }
+    });
+  });
+};
+
+exports.exists = function assertExists(expected, selector) {
+  return new Asserter(function testTarget(target) {
+    if (!!target !== expected) {
+      throw new AssertionError('Element %j should%s exist',
+        selector, (expected ? '' : 'n\'t'));
+    }
+  });
+};
+
+function stringify(value) {
+  if (_.isRegExp(value)) {
+    return '' + value;
+  }
+  return JSON.stringify(value);
+}
+
+function secondToLowerCase(first, second) {
+  return second.toLowerCase();
+}
+
+function getterToPropName(propName) {
+  return propName.replace(/^get([A-Z])/, secondToLowerCase);
```

Comment:
I'm not sure if `getABC` should become `abc`, `aBC`, or `ABC`. I'd consider that "undefined behavior". I don't think that's an actual case this function handles. E.g. all properties it's concerned with are the ones provided by webdriver, e.g. `getText`, `getPageTitle`, etc.